pkg/kubelet/apis/config/v1beta1/defaults_test.go

```diff
@@ -253,6 +254,7 @@ func TestSetDefaultsKubeletConfiguration(t *testing.T) {
 				MemoryThrottlingFactor:          utilpointer.Float64(0),
 				RegisterNode:                    utilpointer.Bool(false),
 				LocalStorageCapacityIsolation:   utilpointer.Bool(false),
+				PodLogsRootDirectory:            "",
```

Comment:
I'd like to see one more test case where you set the `PodLogsRootDirectory` to non default and empty. I know the code will probably work but we should cover that case also.

Reply:
Yes it's reasonable. Done.